### PR TITLE
Mobile zoom

### PR DESCRIPTION
**Pull request by alimtunc**
We couldn't zoom on mobile, I've fixed it by using this plugin https://rexrainbow.github.io/phaser3-rex-notes/docs/site/gesture-pinch/?h=pinch and @xsyann helped me to integrate the rules he added few weeks ago on web zooming, now it's working smoothly 🙌

Closing https://github.com/l3mpire/lemverse/issues/77

### CODE REVIEW

File: core/client/scenes/scene-world.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ WorldScene = new Phaser.Class({
     Phaser.Scene.call(this, { key: 'WorldScene' });
   },
 
+  preload() {
+    const url = 'https://raw.githubusercontent.com/rexrainbow/phaser3-rex-notes/master/dist/rexpinchplugin.min.js';
```
Comment (Donorhan) [inline core/client/scenes/scene-world.js:40]: I wonder if we should not host this file directly in the project. We don't have control over the version and the state of the server. In `client/plugin` for example?

Comment (Donorhan) [reply core/client/scenes/scene-world.js:40]: Wouldn't it be better in the BootScene? The WorldScene is reloaded each time you change levels

Comment (alimtunc) [reply core/client/scenes/scene-world.js]: All done 🙌

Comment (Donorhan) [reply core/client/scenes/scene-world.js]: Thank you!

**Pull request merged**